cloudsql-mysql-plugin/src/main/java/io/cdap/plugin/cloudsql/mysql/CloudSQLMySQLConnectorConfig.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.cloudsql.mysql;
+
+import io.cdap.cdap.api.annotation.Description;
+import io.cdap.cdap.api.annotation.Name;
+import io.cdap.plugin.db.ConnectionConfig;
+import io.cdap.plugin.db.connector.AbstractDBConnectorConfig;
+
+import java.util.Properties;
+import javax.annotation.Nullable;
+
+/**
+ * Configuration for CloudSQL MySQL connector
+ */
+public class CloudSQLMySQLConnectorConfig extends AbstractDBConnectorConfig {
+
+  private static final String JDBC_PROPERTY_CONNECT_TIMEOUT = "connectTimeout";
```

Comment:
just put units in the variable name so it is clear when reading the code, like `JDBC_PROPERTY_CONNECT_TIMEOUT_MILLIS`

Reply:
acknowledged.